t/test-lib.sh

```diff
@@ -37,22 +37,23 @@ then
 fi
 STG_ROOT=$(cd "$TEST_DIRECTORY"/.. && pwd) || exit 1
 export STG_ROOT
-stg_build_dir="$STG_ROOT"/build
 PYTHON="${PYTHON:-python}"
 python_major_minor="$($PYTHON -c '
 import sys
 print(".".join(map(str, sys.version_info[:2])))')"
-stg_bin_dir="$stg_build_dir"/scripts-"$python_major_minor"
+stg_inst_dir="$STG_ROOT/inst/python$python_major_minor"
+stg_bin_dir="$stg_inst_dir"/bin # TODO on Windows this is .../Scripts
```

Comment:
This comment sort of implies that the test suite may not actually be runnable on Windows. What's the status of that?

Reply:
Windows is... difficult :-) Honestly, I forgot about that before making the PR. The whole thing's been sitting on my computer for far too long.